lib/vanagon/component.rb

```diff
@@ -290,6 +290,10 @@ def get_build_dir
       end
     end
 
+    def get_dependency_hash
+      { name => { 'version' => version, 'ref' => options[:ref] }.delete_if { |k, v| !v } }
```

Comment:
Unrelated, I think Rubocop will complain about an unused key in the iterators. It'll ask you to use `_k` or `_` instead of `k`. It's strictly a style nitpick and I wanted to make sure you knew about it if Rubocop hadn't squawked yet.